ironic/drivers/modules/drac/raid.py

```diff
@@ -761,13 +910,27 @@ def delete_configuration(self, task):
         :raises: DracOperationError on an error from python-dracclient.
         """
         node = task.node
+        controllers = list()
+        drac_raid_controllers = list_raid_controllers(node)
+        for cntrl in drac_raid_controllers:
+            if is_raid_controller(node, cntrl.id):
+                controller = dict()
+                controller_cap = reset_config(node, cntrl.id)
+                controller['raid_controller'] = cntrl.id
+                controller['is_reboot_required'] = controller_cap[
+                    'is_reboot_required']
+                controllers.append(controller)
+
+        node_state = _commit_to_controllers(node, controllers)
+        LOG.info("Converting all physical disks to JBOD mode")
+        change_state_result = configure_physical_disk_conversion(node,
+            mode="JBOD")
```

Comment:
On the line above, I think we should pass in dracclient.constants.RaidStatus.jbod